### DISCUSSION TITLE
Indication in playlist that songs originate from one file

### DISCUSSION THREAD

**Discussion by z13swee** [audacious-media-player/audacious#67] [Ideas]
I usually load up audacious with many songs and listen through and remove those i don't like by right click -> Services -> Delete.
The problem is that i might unknowingly delete a file that contains many songs and lose all those songs instead of just the one i wanted to delete.

It would be nice if there was an indication that the song originates from one file, maybe something like this in the playlist:
```
Song
Song
filename(?):
  +-- Song
  +-- Song
  +-- ...
Song
```

Of course this is only a problem with file formats that allows multiple songs in one file, but i thought i mention it.

**Comment by radioactiveman** [comment]
I'm not sure if it is possible for the Delete Files plugin to detect this. @jlindgren90: Do you have an idea?

The plugin has a setting though to use the recycle bin so you can restore the deleted file in this case.

**Comment by jlindgren90** [comment]
The plugin could check if the playlist entry is a subtune (URL ends in question mark + digit) and add an extra warning to the confirmation prompt in that case.
